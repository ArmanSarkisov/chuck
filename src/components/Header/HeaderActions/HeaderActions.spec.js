import { BrowserRouter as Router } from 'react-router-dom';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import { HeaderActions } from './index';

describe('Header actions test', () => {
  let component;

  beforeEach(() => {
    component = mount(
      <Router>
        <HeaderActions />
      </Router>,
    );
  });

  it('should render Joke component', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
