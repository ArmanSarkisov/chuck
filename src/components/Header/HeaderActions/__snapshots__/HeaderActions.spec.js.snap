// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header actions test should render Joke component 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <HeaderActions>
      <div>
        <Button
          onClick={[Function]}
          text="Logout"
        >
          <button
            className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
            onClick={[Function]}
            type="button"
          >
            Logout
          </button>
        </Button>
      </div>
    </HeaderActions>
  </Router>
</BrowserRouter>
`;
